WTWSMS/common/landed_titles/mercenaries.txt

```diff
@@ -2565,4 +2564,80 @@ d_ghanan_band = {
 	strength_growth_per_century = 1.0
```

Comment:
> The two alternatives you yourself present here is that either nothing is done or I decide things unilaterally.

Who died and made you god of this Mod? The Mercenary fixes were desparately needed. Back off and let it happen.

> A golden rule when coding is that anyone should at a glance see what you wrote and understand it.

By your own words, you denying their name changes (etc.) is the **_height_** of hypocrisy.

> We can add a hashtag if you want to.

I OFFERED TO DO THE SAME THING ON "DONAMONAPRISC_DECADENT"! Guess who refused that because of their stupid "golden rule"? -but now it's okay when you do it! You are this Mod's greatest hypocrite!